primo/data_parser/metric_data.py

```diff
@@ -693,8 +691,14 @@ class ImpactMetrics(SetOfMetrics):
 
     def __init__(
         self,
-        impact_metrics: Optional[Dict[str, _SupportedContent]] = SUPP_IMPACT_METRICS,
+        impact_metrics: Optional[Dict[str, _SupportedContent]] = None,
     ) -> None:
+        # Default arguments retain values between function calls.
+        # If we use a mutable argument, unexpected things can occurs.
+        # See this for example:
+        # https://stackoverflow.com/questions/26320899/
+        if impact_metrics is None:
```

Comment:
This post makes sense, but highlights that it is only a problem if the default argument is modified by the function. This is an input that is never modified, so I think we are safe. However, I don't see the change doing any harm.

Reply:
Yep, just trying to avoid issues in the future if we make some changes to that...